apps/photos/src/services/HTTPService.ts

```diff
@@ -15,12 +17,38 @@ class HTTPService {
     constructor() {
         axios.interceptors.response.use(
             (response) => Promise.resolve(response),
-            (err) => {
-                if (!err.response) {
-                    return Promise.reject(err);
+            (error) => {
+                if (error.response) {
+                    // The request was made and the server responded with a status code
+                    // that falls out of the range of 2xx
+                    logError(error, 'HTTP Service Error', {
```

Comment:
Will this log on sentry? If yes, do we really want it for all requests?

Reply:
yes, it will be logged on sentry and No harm in having them no?...will help with debugging?

Reply:
Hmm, I feel it would increase the noise, but could be wrong as well. You would know better.